src/server.c

```diff
@@ -5885,7 +5940,12 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "io_threaded_reads_processed:%lld\r\n"
             "io_threaded_writes_processed:%lld\r\n"
             "reply_buffer_shrinks:%lld\r\n"
-            "reply_buffer_expands:%lld\r\n",
+            "reply_buffer_expands:%lld\r\n"
+            "eventloop_cycles:%llu\r\n"
```

Comment:
@CharlesChen888 @oranagra I think we forget to document this on the info documentation page. Unrelated, reply_buffer_expands is also not documented, I wonder if there are others not documented as well.

Reply:
Just raised a PR: redis/redis-doc#2439